cardano-cli/cardano-cli.cabal

```diff
@@ -162,3 +161,34 @@ test-suite cardano-cli-test
                         -Wpartial-fields
                         -Wcompat
                         -threaded -rtsopts -with-rtsopts=-N -with-rtsopts=-T
+
+test-suite cardano-cli-scenarios
+  hs-source-dirs:       test
+  main-is:              cardano-cli-scenarios.hs
+  type:                 exitcode-stdio-1.0
+
+  if os(windows)
+    buildable: False
```

Comment:
Is it true that this isn't buildable on Windows? I don't think we're doing anything *nix specific here, right?